core/client/verify_test.go

```diff
@@ -1,4 +1,4 @@
-// Copyright 2017 Google Inc. All Rights Reserved.
+// Copyright 2018 Google Inc. All Rights Reserved.
```

Comment:
Not sure it's worthwhile updating (c) years in existing files.

Reply:
yeah, I don't know what the guidance on this is.  Some people recommend updating, some don't. I don't know whether it's better to have newer or older dates. 

My current strategy is to update the dates when I significantly rework files.